client/src/components/styledText.css

```diff
@@ -95,6 +96,16 @@
     filter: drop-shadow(0 0 .2em var(--spiral-shine))
             drop-shadow(.1rem .1rem .1rem rgb(0, 0, 0));
 }
+.keyword-poly {
+    /*
+        Not scrolling because I looked it up and one of the reasons pi is on there is that is pi is greek letter for p so p should be yellow
+        Gradient is massive because it makes it stop being a gradient and makes the number of each color allows the color to line up with the character near exactly (only not exact because kerning means that the L & Y overlap)
+    */
```

Comment:
I don't think you should need a comment here. CSS has a way to do what you want without being as obtuse as this. Look at what I did for Santa Claus